tests/test_parser.py

```diff
@@ -175,6 +220,35 @@ def test_with_unknown_args(self):
         self.assertEqual(args.gmp_password, 'foo')
         self.assertEqual(script_args, ['--bar', '--bar2'])
 
+    @mock.patch('gvmtools.parser.logging')
+    def test_socket_has_no_timeout(
+        self, logging_mock
```

Comment:
Just use an underscore to indicate and unused argument. Pylint will ignore it with underscore.
```suggestion
        self, _logging_mock
```